src/applications/edu-benefits/10216/components/PercentageCalc.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { calculatedPercentage } from '../utilities';
+
+const gaRatioCalculationText = () => {
+  window.dataLayer.push({
+    event: 'edu-0994--form-help-text-clicked',
+    'help-text-label': 'How is this calculated?',
+  });
+};
+function PercentageCalc({ formData }) {
+  return (
+    <div className="schemaform-field-template">
+      <span className="vads-u-font-weight--bold vads-u-font-size--lg">
+        {calculatedPercentage(formData)}
+      </span>
+      <va-additional-info
+        class="vads-u-margin-top--1p5"
+        trigger="How is this calculated?"
+        onClick={gaRatioCalculationText}
+      >
+        <span>
```

Comment:
wrapping in span vs fragment might add some padding-left here we don't want, please check